README.md

```diff
@@ -39,6 +39,18 @@ but allows developers to run the Python code on their native system.
    2. `celery worker --app dandi.celery --loglevel info --without-heartbeat`
 2. When finished, run `docker-compose stop`
 
+## Connecting to `dandiarchive`
+`dandiarchive` is another application and will need to be setup and run separately. 
+1. Login to the `dandiarchive` Girder client using the `publish` admin account. If 
+you followed the README it will be located at `http://localhost:8080/`.
+   * **NOTE**: the username of the Girder admin account used here must be `publish` for publishing to work properly. If an admin account with that username doesn't exist, it must be created.
+2. Navigate to account settings, click on the 'API keys' tab, and generate an API key.
+3. Save this API key and the Girder client URL in environment variables named `DJANGO_DANDI_GIRDER_API_KEY`
```

Comment:
It might be good to also document how to do this (i.e. edit the appropriate `dev/.env*` file).

Reply:
The use of the `dev/.env*` files are slightly changed in #55 - will that be merged in the near future? If so, I can describe how to edit the `dev/.env*` files in that branch and hold off on merging this PR until that one is merged, if not I can add instructions for editing the `dev/.env*` files in `master` as it is right now.